internal/loadbalancers/api/v1/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Compiled Object files, Static and Dynamic libs (Shared Objects)
```

Comment:
The internal client in CPEM was made to hit a deadline.  We don't have that time pressure here, so this should use a generated client that lives in [equinix-sdk-go](https://github.com/equinix/equinix-sdk-go)